### PR TITLE
add more infos to delete Popup

### PR DESCRIPTION
**Pull request by T0biii**
We could talk about the text, but I think some more information about what will be deleted would be good

User:
image

Device:
image

### CODE REVIEW

File: website/src/pages/admin/AllDevices.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const AllDevices = observer(class AllDevices extends React.Component {
   });
 
   deleteUser = async (user: User.AsObject) => {
-    if (await confirm('Are you sure?')) {
+    if (await confirm('Delete ' + user.name + ' are you sure? ')) {
```
Comment (GoliathLabs) [inline website/src/pages/admin/AllDevices.tsx:36]: 'Are you sure you want to delete ' + user.name + '?'
would be a better fit

File: website/src/pages/admin/AllDevices.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export const AllDevices = observer(class AllDevices extends React.Component {
   }
 
   deleteDevice = async (device: Device.AsObject) => {
-    if (await confirm('Are you sure?')) {
+    if (await confirm('Delete ' + device.name + ' from ' + device.ownerName + ' are you sure?')) {
```
Comment (GoliathLabs) [inline website/src/pages/admin/AllDevices.tsx:46]: Are you sure you want to delete ' + device.name + ' from ' + device.ownerName + '?'

**Review by GoliathLabs — CHANGES_REQUESTED** [review]

File: website/src/pages/admin/AllDevices.tsx (hunk 2 above)
Comment (T0biii) [reply website/src/pages/admin/AllDevices.tsx]: image

File: website/src/pages/admin/AllDevices.tsx (hunk 1 above)
Comment (T0biii) [reply website/src/pages/admin/AllDevices.tsx]: image

**Review by GoliathLabs — APPROVED** [review]

**Pull request merged**